### ISSUE TITLE
 FailedAttachVolume for app

### ISSUE THREAD

**Issue by kyberorg** [acheaito/nextcloud-kubernetes#2]
Given setup leads to FailedMount

  | Multi-Attach error for volume  "pvc-1209d273-a666-441d-afee-37c013f3caa1" Volume is already used by  pod(s) nextcloud-cron-54fb8577cf-xzsl9
-- | --

It can be fixed by using PVC in ReadWriteMany (RWX) mode

**Comment by cleitonpena** [comment]
Yes, @kyberorg I also changed it to RWX. I'm using Openebs cStor.
However, I tried in every way to change the php.in settings, in this case .user.ini and .htacess, and I was not successful in uploading files with more than 1MB.
Were you able to upload more than 1MB?
